protocol/rpcprovider/reliabilitymanager/reliability_manager.go

```diff
@@ -104,7 +105,7 @@ func (rm *ReliabilityManager) VoteHandler(voteParams *VoteParams, nodeHeight uin
 		// we need to send a commit, first we need to use the chainProxy and get the response
 		// TODO: implement code that verified the requested block is finalized and if its not waits and tries again
 		ctx := context.Background()
-		chainMessage, err := rm.chainParser.ParseMsg(voteParams.ApiURL, voteParams.RequestData, voteParams.ConnectionType, nil, 0)
+		chainMessage, err := rm.chainParser.ParseMsg(voteParams.ApiURL, voteParams.RequestData, voteParams.ConnectionType, nil, extensionslib.ExtensionInfo{LatestBlock: 0}) // TODO: do we have the latest block?
```

Comment:
we need to do overwrite extensions here from the vote information of extensions

Reply:
added todo